developer/js/package.json

```diff
@@ -36,7 +36,7 @@
   },
   "dependencies": {
     "commander": "^3.0.0",
-    "node": "^11.7.0",
+    "node": "^12.0.0",
```

Comment:
I believe this change should be in the `engines` key in `package.json`, not in dependencies See: https://docs.npmjs.com/files/package.json#engines

Reply:
Okay, I've added the `engines` key but also left this one the same. I don't think that's a problem, right?

Reply:
It's not a problem, _per se_, but the `node` npm package is [an actual real package](https://www.npmjs.com/package/node) and all it does is it downloads a binary of node for the current platform. It doesn't declare a dependency on node itself; it _installs_ a version of node! Try ls'ing `node_modules/node/bin`.